src/libguac/guacamole/rwlock.h

```diff
@@ -82,15 +82,15 @@ typedef struct guac_reentrant_rwlock {
  * @param lock
  *     The guac reentrant rwlock to be initialized.
  */
-void guac_init_reentrant_rwlock(guac_reentrant_rwlock* lock);
+void guac_init_rwlock(guac_rwlock* lock);
 
 /**
  * Clean up and destroy the provided guac reentrant rwlock.
  *
  * @param lock
  *     The guac reentrant rwlock to be destroyed.
  */
-void guac_destroy_reentrant_rwlock(guac_reentrant_rwlock* lock);
+void guac_destroy_rwlock(guac_rwlock* lock);
```

Comment:
To match convention, these should be `guac_rwlock_init()` and `guac_rwlock_destroy()` respectively.